community.md

```diff
@@ -48,33 +48,22 @@ We run an announcement-only, no-marketing-spam mailing list that we will use to
 <input class="cTextfieldstyle" maxlength="90" value="" id="emailUserOS" name="email" placeholder="I consent to join the email list" title="Email" type="text">
 </form> -->
 
-## Want to report issues? 
+## How you can help 
 
-Oh no, you hit a bump on the road? Please tell us - we want to fix all bugs!
+Getting a language established is like getting a new religion going. We really need you, our fans, to help spread the word.
 
-- Found a bug in the compiler, runtime, standard library, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
-- Found a bug in the language specification? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
-- Found a bug on the website? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
-- Found a security flaw? Send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
+- Start with starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!

```

Comment:
Instead of main repo, shall we rephrase it as Ballerina Github repo?

Reply:
Shouldn't this be
```suggestion
- Start by starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!
```

Reply:
> Instead of main repo, shall we rephrase it as Ballerina Github repo?

Done